modules/acquia_cms_common/composer.json

```diff
@@ -18,7 +18,6 @@
         "drupal/checklistapi": "^2.0",
         "drupal/config_ignore": "2.3.0",
         "drupal/config_rewrite": "^1.4",
-        "drupal/core": "^9.3.21",
```

Comment:
@chandan-singh7929 Why this is removed ? How are we making sure that our module works with Drupal Core `^9.3.x`

Reply:
acquia_cms_common module is compatible to `^9 || ^10` as per .info.yml hence not required to mention in composer.json.
If there is any specific function or code which deprecate in earlier versions then compatibility needs to be updated to `^9.3 || ^10` and no need to mention in composer.json.

Reply:
Done, updated info to have `core_version_requirement: ^9.3 || ^10`